boards/nordic/nrf52840_dongle/src/main.rs

```diff
@@ -119,6 +120,15 @@ impl kernel::Platform for Platform {
 pub unsafe fn reset_handler() {
     // Loads relocations and clears BSS
     nrf52840::init();
+    let ppi = static_init!(nrf52840::ppi::Ppi, nrf52840::ppi::Ppi::new());
+    // Initialize chip peripheral drivers
+    let nrf52840_peripherals = static_init!(Nrf52840Peripherals, Nrf52840Peripherals::new(ppi));
+
+    // set up circular peripheral dependencies
+    {
```

Comment:
I don't see why this pair of braces is needed.

Reply:
thanks, fixed